nextflow.config

```diff
@@ -31,6 +36,19 @@ params {
     ref_flat                     = "${params.dx_tracks_path}/rna/${params.gencode_version_name}.ref_annot.gtf.refflat"
     rrna_intervals               = "${params.dx_tracks_path}/rna/GRCh38_rRNA_genbank.interval_list"
 
+
+    // Reference sets for outrider
+
+    outrider_gene_chx_parents   = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/chx/gene_set1_parents_20250702/"
+    outrider_gene_chx_index     = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/chx/gene_set2_index_20250702/"
+    outrider_gene_cntrl_parents = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/cntrl/gene_set1_parents_20250702/"
+    outrider_gene_cntrl_index   = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/cntrl/gene_set2_index_20250702/"
+
+    outrider_exon_chx_parents   = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/chx/exon_set1_parents_20250702/"
+    outrider_exon_chx_index     = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/chx/exon_set2_index_20250702/"
+    outrider_exon_cntrl_parents = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/cntrl/exon_set1_parents_20250702/"
+    outrider_exon_cntrl_index   = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/cntrl/exon_set2_index_20250702/"
```

Comment:
I am a bit worried about reproducibility and having some approach that is future proof. 

Is it possible to log more information regarding the reference files?
Which samples are included? md5sum of files? All same version of workflow? (same alignment, featurecounts etc.)

Is comparing chx + cntrl something UMC specific? Do we need a mechanism to skip outrider?